sql/addons_v3.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
+  @submission_date AS submission_date,
+  document_id,
+  client_id,
+  sample_id,
+  subsession_start_date,
+  normalized_channel,
+  _addon.element.*
+FROM
+  telemetry.main_summary_v4,
+  UNNEST(active_addons.list) AS _addon
+WHERE
+  submission_date_s3 = @submission_date
+  AND client_id IS NOT NULL
```

Comment:
based on the discussion [here](https://github.com/mozilla/bigquery-etl/pull/41#discussion_r269596130) I think we should switch these clauses to ease backfill:

```diff
-  submission_date_s3 = @submission_date
-  AND client_id IS NOT NULL
+  client_id IS NOT NULL
+  AND submission_date_s3 = @submission_date
```